l10n/pl/add-on.properties

```diff
@@ -69,50 +68,28 @@ settings_menu_item_turnnotificationsoff=Wyłącz powiadomienia
 settings_menu_item_feedback=Prześlij swoją opinię…
 settings_menu_button_tooltip=Ustawienia
 
-initiate_call_button_label2=Rozpocząć rozmowę?
-incoming_call_title2=Rozmowa przychodząca
-incoming_call_block_button=Zablokuj
-hangup_button_title=Rozłącz
-hangup_button_caption2=Zakończ rozmowę
-
-call_with_contact_title=Rozmowa z {{incomingCallIdentity}}
-
-outgoing_call_title=Rozpocząć rozmowę?
-initiate_audio_video_call_button2=Rozpocznij
-initiate_audio_video_call_tooltip2=Rozpocznij (dźwięk i obraz)
-initiate_audio_call_button2=Rozpocznij (tylko dźwięk)
-
-peer_ended_conversation2=Rozmówca się rozłączył.
-restart_call=Połącz ponownie
-
-contact_offline_title=Kontakt offline
-call_timeout_notification_text=Nieudane połączenie.
-
-cancel_button=Anuluj
 rejoin_button=Połącz ponownie
 
-cannot_start_call_session_not_ready=Nie można rozpocząć rozmowy, sesja nie jest gotowa.
-network_disconnected=Połączenie sieciowe niespodziewanie zakończone.
-connection_error_see_console_notification=Nieudane połączenie (szczegóły można znaleźć w konsoli).
 no_media_failure_message=Nie znaleziono kamery ani mikrofonu.
 ice_failure_message=Nieudane połączenie. Zapora sieciowa może blokować połączenia.
 
 legal_text_and_links3=Używając {{clientShortname}}, zgadzasz się na {{terms_of_use}} oraz {{privacy_notice}}.
 legal_text_tos=warunki korzystania z usługi
 legal_text_privacy=politykę prywatności
+
+loop_deprecate_notice_and_link=Firefox Hello niedługo zostanie usunięty z Firefoksa. {{learn_more}}
+loop_deprecate_learn_more=Więcej informacji
```

Comment:
@splewako: please review these two new strings.

The original strings: https://github.com/mozilla/loop-client-l10n/blob/master/l10n/templates/add-on.properties#L132